src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -65,7 +65,7 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
   }
 
   majorVersion(): number {
-    return parseInt(this.version().split('.')[0]);
+    return this.version() !== null ? parseInt(this.version().split('.')[0]) : -1;
```

Comment:
Is there any possibility that `this.version` might return `undefined`?

Reply:
How about returning `-1` for the `undefined` version too?

```js
typeof this.version() === 'string' ? parseInt(this.version().split('.')[0]) : -1;
```

Reply:
According to `detect-browser` library, no. It could only be `string` or `null`. But falsy check is better than null check here. Could we update this in later if needed?

Reply:
Thanks for checking!